src/v2/view-builder/components/AuthenticatorEnrollOptions.js

```diff
@@ -24,7 +24,7 @@ const AuthenticatorRow = View.extend({
         <div class="authenticator-description">\
           <h3 class="authenticator-label no-translate">{{label}}</h3>\
           {{#if description}}\
-            <p class="authenticator-description--text">{{description}}</p>\
+            <p class="authenticator-description--text no-translate">{{description}}</p>\
```

Comment:
Shouldn't this `description` point to what is defined [here](https://github.com/okta/okta-signin-widget/blob/master/src/v2/view-builder/utils/AuthenticatorUtil.js#L75-L77)? Seems like we **always** want these to be localized.

Reply:
yeah, it is not just the authenticator description. In some cases we also have a phone number which is what I was trying to ignore in this case.

![Screen Shot 2021-06-15 at 12 16 39 PM](https://user-images.githubusercontent.com/23267876/122110706-b72e7200-cdd3-11eb-809f-9ac4722b255b.png)

Reply:
Its for the phone number cases https://github.com/okta/okta-signin-widget/blob/master/src/v2/view-builder/utils/AuthenticatorUtil.js#L55

Reply:
@jmelberg-okta updated it to add the no-translate className conditionally.